app/src/main/java/com/nearsoft/referralsapp/job_details/JobDetailsActivity.java

```diff
@@ -0,0 +1,50 @@
+package com.nearsoft.referralsapp.job_details;
+
+import android.os.Build;
+import android.os.Bundle;
+import android.support.annotation.RequiresApi;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+
+import com.nearsoft.referralsapp.R;
+
+import java.util.ArrayList;
+import java.util.Objects;
+
+public class JobDetailsActivity extends AppCompatActivity {
+    private ArrayList<String> mJobDescription = new ArrayList<>();
+    private ArrayList<String> mTitle;
+
+    @RequiresApi(api = Build.VERSION_CODES.KITKAT)
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.job_details_activity);
+
+        FillDescription();
```

Comment:
It seems like fillDescription could instatiate and return also the JobDetailAdapter and turn mJobDescription and mTitle local to fillDescription